### PR TITLE
[NUI][NUI.Components.Devel] Fix failed isseus of NUI.Components.Devel.

### PR DESCRIPTION
**Pull request by AchoWang**
### Description of Change ###
Fix failed isseus of NUI.Components.Devel.

### API Changes ###

 - ACR:

### CODE REVIEW

File: test/Tizen.NUI.Tests/Tizen.NUI.Components.Devel.Tests/Properties/Resources.resx [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-    The primary goals of this format is to allow a simple XML format
-    that is mostly human readable. The generation and parsing of the
-    various data types are done through the TypeConverter classes
+    
+    The primary goals of this format is to allow a simple XML format 
+    that is mostly human readable. The generation and parsing of the 
+    various data types are done through the TypeConverter classes 
     associated with the data types.
-
+
```
Comment (zhouleonlei) [inline test/Tizen.NUI.Tests/Tizen.NUI.Components.Devel.Tests/Properties/Resources.resx:12]: Please remove the spaces

**Review by xuelian-bai — APPROVED** [review]
looks good to me

**Review by wanchao-xu — APPROVED** [review]
Looks good to me

**Review by zhouleonlei — APPROVED** [review]
lgtm

**Review by wb0716 — APPROVED** [review]
Good for me

**Review by xiaowei-guan — APPROVED** [review]
LGTM

**Review by elishateng — APPROVED** [review]
LGTM

**Pull request closed without merging**